src/IssueViz/IssueViz.csproj

```diff
@@ -11,7 +11,7 @@
   <PropertyGroup>
     <TargetFramework>net46</TargetFramework>
     <ProjectGuid>{94130D79-3F30-47B1-9C3F-0769E57D1F4D}</ProjectGuid>
-  <RootNamespace>SonarLint.VisualStudio.IssueVisualization</RootNamespace>
+    <RootNamespace>SonarLint.VisualStudio.IssueVisualization</RootNamespace>
```

Comment:
The root namespace is set twice, presumably as the result of an automatic merge.